crates/cheatcodes/src/impls/evm.rs

```diff
@@ -305,6 +305,21 @@ impl Cheatcode for revertToCall {
     }
 }
 
+impl Cheatcode for loadAllocsCall {
+    fn apply_full<DB: DatabaseExt>(&self, ccx: &mut CheatsCtxt<DB>) -> Result {
+        // First, load the allocs file from the provided path.
+        let file = File::open(&self.pathToAllocsJson)?;
+        let allocs: HashMap<Address, GenesisAccount> = serde_json::from_reader(file)?;
+
+        // Then, load the allocs into the database.
+        ccx.data
+            .db
+            .load_allocs(&allocs, &mut ccx.data.journaled_state)
+            .map(|_| Vec::default())
+            .map_err(|e| fmt_err!("Failed to load allocs: {e}"))
+    }
```

Comment:
can you please keep style consistent with the rest of the cheatcodes here? `Self` destructure at the top and lowercase error message.